docs/_content/standaard/catalogi/index.md

```diff
@@ -118,6 +118,28 @@ parametrisering is aan validatieregels onderhevig:
     * MOET een waarde hebben indien de afleidingswijze `termijn` is
     * MAG GEEN waarde hebben in de andere gevallen
 
+#### **<a name="ztc-009">Concepten ([ztc-009](#ztc-009))</a>**
+
+De resources `Zaaktype`, `InformatieObjecttype` en `Besluittype` bevatten het veld `concept`, indien dit veld aangemerkt is als `True`, dan betreft het een niet-definitieve versie van het object die niet buiten de Catalogi API gebruikt mag worden.
+
+Om het object te veranderen naar een definitieve versie, kan het object via de API gepubliceerd worden. Zodra een object gepubliceerd is, staat het veld `concept` op `False`.
```

Comment:
Hoewel het aan de werking niets afdoet komt het op mij een beetje gekunsteld over dat het attribuut "concept" gezet wordt door de actie "publiceren". Is het dan niet logischer om het attribuut "gepubliceerd" oid te noemen? Dan is ook meteen duidelijk dat een zaaktype wat geen concept (meer) is maar ook niet gepubliceerd is niet gebruikt mag worden buiten de catalogi api.